### ISSUE TITLE
Neural Networks in a Nutshell

### ISSUE THREAD

**Issue by ParthikB** [pydatadelhi/talks#107]
- Abstract (2-3 lines)
Machine Learning and Artificial Intelligence are today's buzzwords. And the one that is emerging as a hero is Deep Learning, which is nothing but a Deep Neural Network. Being thorough with a basic Neural Network can strengthen one's base before diving into Deep Learning. 

- Brief Description and Contents to be covered
A general introduction to Deep Learning and Neural Networks will be given from the most basics so that a layman can even follow up. Contents I'm planning to cover:
1) What is a Neuron
2) Basic Neural Network Structure
3) Forward Propagation
4) Cost Function
5) Backpropagation (just a shallow introduction)
6) Gradient Descent (just a shallow introduction)
7) Building a basic Neural Network from Scratch (hands-on) without using any ML Library. 

- Pre-requisites for the talk
1) Should've at least a little idea about Machine Learning.
2) Should've worked with Python (especially numpy library)

- Time required for the talk
A thorough introduction might take an hour (including hands-on), but if needed, can be wrapped up within 30 minutes.

- Link to slides
Not yet made. I will make sure to make it if I'm given this opportunity. Though I've got a blog prepared on it whose link is given within the below repository.

- Will you be doing hands-on demo as well?
Yes

- Link to ipython notebook (if any)
https://github.com/ParthikB/Neural-Networks-from-Scratch-2/tree/master/Deep%20Neural%20Networks

- About yourself
 I'm a student currently pursuing Engineering sadly not in Computer discipline but my love for the advancing technologies has bought me in the arena of Artificial Intelligence and Machine Learning. I'm a determined Machine Learning enthusiast, specifically Deep Learning, who's working his way up this ladder with each passing hour in parallel with the Electronics discipline I'm enrolled in. In a nutshell, I'm a friend first, determined second and probably entertainer third.!

- Are you comfortable if the talk is recorded and uploaded to PyData Delhi's YouTube channel ?
Yeah sure.

- Any query ?
I just want to know if I'll be given a choice of dates to give the presentation on or I've to do it in whichever meetup is the nearest? Also, I can also give a lightning talk on "Live Plotting with Matplotlib".
